src/registrar/tests/test_views.py

```diff
@@ -1500,3 +1500,19 @@ def test_application_status_no_permissions(self):
                         reverse(url_name, kwargs={"pk": application.pk})
                     )
                     self.assertEqual(page.status_code, 403)
+
+
+def test_approved_application_not_in_active_requests(self):
```

Comment:
**comment**: It looks like this probably crept in from a separate workstream. Is it easy enough leave this change out of this PR?